src/org/mozilla/javascript/MemberBox.java

```diff
@@ -21,49 +21,32 @@
 {
     static final long serialVersionUID = 6358550398665688245L;
 
-    private transient Member memberObject;
-    transient Class<?>[] argTypes;
-    transient Object delegateTo;
-    transient boolean vararg;
+    private transient Executable memberObject;
```

Comment:
Can we make this final? I'm asking because some of the blogs talk about JVM optimizations for reflection (and MethodHandles, which we don't use yet) that only happen when they're final.

Reply:
Unfortunately, it can't be both transient and final as then it couldn't be assigned upon deserialization.